framework/sandstone.h

```diff
@@ -293,6 +293,9 @@ typedef enum test_flag {
     /// Indicates that a test can only attribute failure to a particular
     /// package and not to threads or cores.
     test_failure_package_only       = 0x1000,
```

Comment:
on a side note, why `test_failure_package_only` and `test_flag_ignore_do_while` have the same  value?

Reply:
I should have worn glasses. 🤣